diff -up lynx2.9.2/lynx.cfg.2~ lynx2.9.2/lynx.cfg
--- lynx2.9.2/lynx.cfg.2~	2024-06-05 15:06:13.064104063 +0200
+++ lynx2.9.2/lynx.cfg	2024-06-05 15:06:56.840571324 +0200
@@ -1,7 +1,8 @@
 # $LynxId: lynx.cfg,v 1.358 2024/05/31 22:09:16 tom Exp $
 # lynx.cfg file.
-# The default placement for this file is /usr/local/lib/lynx.cfg (Unix)
-#                                     or Lynx_Dir:lynx.cfg (VMS)
+#
+# Please don't edit this file directly (it is updated with every OpenMandriva
+# Lx update, overwriting your changes). Instead, edit /etc/lynx-site.cfg.
 #
 # LYNX_VERSION "2.9.2"
 # LYNX_DATE    "31 May 2024"
@@ -105,7 +106,7 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:https://lynx.invisible-island.net/
+STARTFILE:https://openmandriva.org/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
@@ -129,7 +130,8 @@ STARTFILE:https://lynx.invisible-island.
 # This definition will be overridden if the "LYNX_HELPFILE" environment
 # variable has been set.
 #
-HELPFILE:https://lynx.invisible-island.net/lynx_help/lynx_help_main.html
+#HELPFILE:https://lynx.invisible-island.net/lynx_help/lynx_help_main.html
+HELPFILE:file://localhost/usr/share/lynx_help/lynx_help_main.html
 .ex
 #HELPFILE:file://localhost/PATH_TO/lynx_help/lynx_help_main.html
 
@@ -575,7 +577,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # Older browser versions will fail so keep NCR_IN_BOOKMARKS:FALSE if you
 # plan to use them.
 #
-#NCR_IN_BOOKMARKS:FALSE
+NCR_IN_BOOKMARKS:TRUE
 
 .h2 FORCE_8BIT_TOUPPER
 # FORCE_8BIT_TOUPPER overrides locale settings and uses internal 8-bit
@@ -1811,6 +1813,8 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #
 # Printer examples:
 .ex 3
+PRINTER:Print to default printer:lp %s:TRUE
+PRINTER:View with less:less %s:TRUE
 #PRINTER:Computer Center printer:lpr -Pccprt %s:FALSE
 #PRINTER:Office printer:lpr -POffprt %s:TRUE
 #PRINTER:VMS printer:print /queue=cc$print %s:FALSE:58
@@ -1854,6 +1858,9 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # The following is strongly recommended to undo ASCII->EBCDIC conversion.
 .ex
 #DOWNLOADER:Save OS/390 binary file: iconv -f IBM-1047 -t ISO8859-1 %s >%s:FALSE
+DOWNLOADER:View with less:less %s:TRUE
+DOWNLOADER:View with zless:zless %s:TRUE
+DOWNLOADER:View with xzless:xzless %s:TRUE
 
 .h1 Interaction
 
@@ -2307,7 +2314,7 @@ MINIMAL_COMMENTS:TRUE
 # XLOADIMAGE_COMMAND definition.
 # If you use xli, you may want to add the -quiet flag.
 #
-#XLOADIMAGE_COMMAND:xli %s &
+XLOADIMAGE_COMMAND:display %s &
 
 .h2 VIEWER
 # MIME types may be assigned to external viewers using
@@ -2352,13 +2359,15 @@ MINIMAL_COMMENTS:TRUE
 # by entries in those files.
 #
 .ex 7
-#VIEWER:application/postscript:ghostview %s&:XWINDOWS
-#VIEWER:image/gif:xli %s&:XWINDOWS
-#VIEWER:image/x-xbm:xli %s&:XWINDOWS
-#VIEWER:image/png:xli %s&:XWINDOWS
-#VIEWER:image/tiff:xli %s&:XWINDOWS
-#VIEWER:image/jpeg:xli %s&:XWINDOWS
-#VIEWER:video/mpeg:mpeg_play %s &:XWINDOWS
+VIEWER:application/postscript:okular %s&:XWINDOWS
+VIEWER:application/pdf:okular %s&:XWINDOWS
+VIEWER:image/gif:display %s&:XWINDOWS
+VIEWER:image/x-xbm:display %s&:XWINDOWS
+VIEWER:image/png:display %s&:XWINDOWS
+VIEWER:image/tiff:display %s&:XWINDOWS
+VIEWER:image/jpeg:display %s&:XWINDOWS
+VIEWER:video/mpeg:mpv %s &:XWINDOWS
+VIEWER:text/plain:less %s &
 
 .h2 GLOBAL_MAILCAP
 .h2 PERSONAL_MAILCAP
@@ -3860,3 +3869,5 @@ COLOR:6:brightred:black
 .h2 LIST_DECODED
 # For -dump, show URL-encoded links decoded.
 #LIST_DECODED:TRUE
+# included last to allow overriding all of the configuration above
+INCLUDE:/etc/lynx-site.cfg
