diff -up lynx2.9.1/src/LYMain.c.4~ lynx2.9.1/src/LYMain.c
--- lynx2.9.1/src/LYMain.c.4~	2024-03-18 00:10:41.000000000 +0100
+++ lynx2.9.1/src/LYMain.c	2024-04-19 14:25:16.724626268 +0200
@@ -1047,6 +1047,7 @@ int main(int argc,
     struct stat dir_info;
     char filename[LY_MAXPATH];
     BOOL LYGetStdinArgs = FALSE;
+    struct stat st;
 
 #if USE_GETPWUID
     struct passwd *my_pwd;
@@ -1305,6 +1306,21 @@ int main(int argc,
 #endif
 	LYTildeExpand(&lynx_temp_space, TRUE);
 
+
+    if (stat(lynx_temp_space, &st) < 0) {
+	    if (errno = ENOENT) {
+		    printf("Creating dir %s\n", lynx_temp_space);
+		    mkdir(lynx_temp_space, 0700);
+	    }
+    }
+
+    if (stat(lynx_temp_space, &st) < 0) {
+	    if (errno = ENOENT) {
+		    printf("Can't create %s, let's try /tmp/\n", lynx_temp_space);
+		    lynx_temp_space = "/tmp/";
+	    }
+    }
+    
     if ((cp = strstr(lynx_temp_space, "$USER")) != NULL) {
 	char *cp1;
 
diff -up lynx2.9.1/userdefs.h.4~ lynx2.9.1/userdefs.h
--- lynx2.9.1/userdefs.h.4~	2024-04-19 14:21:31.934573257 +0200
+++ lynx2.9.1/userdefs.h	2024-04-19 14:21:32.002573874 +0200
@@ -366,7 +366,7 @@
  * the "TMPDIR" (unix), or "TEMP" or "TMP" (Windows,DOS,OS/2)
  * variable.
  */
-#define TEMP_SPACE "/tmp/"
+#define TEMP_SPACE "~/tmp/"
 
 /********************************
  * Comment this line out to disable code that implements command logging
