diff -up lynx2.8.9rel.1/lynx.cfg.0000~ lynx2.8.9rel.1/lynx.cfg
--- lynx2.8.9rel.1/lynx.cfg.0000~	2018-09-16 08:40:14.804100399 +0200
+++ lynx2.8.9rel.1/lynx.cfg	2018-09-16 08:42:52.269988430 +0200
@@ -13,7 +13,7 @@
 #	VARIABLE:DEFINITION
 # NO spaces are allowed around the colon ":" between the pair items.
 #
-#  If you do not have write access to /usr/local/lib you may change
+#  If you do not have write access to /usr/bin you may change
 #  the default location of this file in the userdefs.h file and recompile,
 #  or specify its location on the command line with the "-cfg"
 #  command line option.
diff -up lynx2.8.9rel.1/userdefs.h.0000~ lynx2.8.9rel.1/userdefs.h
--- lynx2.8.9rel.1/userdefs.h.0000~	2018-07-08 17:22:44.000000000 +0200
+++ lynx2.8.9rel.1/userdefs.h	2018-09-16 08:40:14.803100393 +0200
@@ -119,8 +119,8 @@
  * Mappings in these global and personal files override any SUFFIX
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_EXTENSION_MAP "Lynx_Dir:mime.types"
-#define PERSONAL_EXTENSION_MAP "mime.types"
+#define GLOBAL_EXTENSION_MAP "/etc/mime.types"
+#define PERSONAL_EXTENSION_MAP ".mime.types"
 
 /**************************
  * The MAILCAP file allows you to map file MIME types to
@@ -129,7 +129,7 @@
  * Mappings in these global and personal files override any VIEWER
  * definitions in lynx.cfg and built-in defaults from src/HTInit.c.
  */
-#define GLOBAL_MAILCAP "Lynx_Dir:mailcap"
+#define GLOBAL_MAILCAP "/etc/mailcap"
 #define PERSONAL_MAILCAP ".mailcap"
 
 /**************************
